pkg/types/vsphere/validation/platform.go

```diff
@@ -94,3 +100,18 @@ func validateFolder(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+// ValidateVCenterAddress validates vCenter to only contain hostname and port
+func ValidateVCenterAddress(vCenter string) error {
+	if strings.TrimSpace(vCenter) == "" {
+		return errors.New("must specify the name of the vCenter")
+	}
```

Comment:
I don't think this check is necessary. vCenter is a required field so it should be checked before this. try removing and see what happens.

Reply:
When I give something like "       " in vCenter name it is read a string of spaces and fails. I have added a validation.